modeling/model_utils/modeling.py

```diff
@@ -0,0 +1,80 @@
+import logging.config
+from typing import List, Dict
+
+import pandas as pd
+from sklearn.model_selection import (
+    train_test_split,
+    cross_validate,
+    GridSearchCV,
+)
+from sklearn.pipeline import Pipeline
+
+from model_utils.constants import LOG_CONFIG_PATH
+
+logging.config.fileConfig(fname=LOG_CONFIG_PATH, disable_existing_loggers=False)
+logger = logging.getLogger(__name__)
+
+
+def load_data(read_path: str, X_col: List[str], y_col: str):
```

Comment:
we have read training data function in pipeline_io already, would this be used anywhere?

Reply:
yea this is just in case we are not reading the train data. Maybe we want to do some feature engineering etc and this makes it flexible to read any path